CONTRIBUTING.md

```diff
@@ -375,6 +376,27 @@ The circleci uses pytest-split to balance unittest load on multiple nodes. In or
 CI_PYTEST_SPLIT_ARGS="--store-durations" tox -e py37
 ```
 
+### Functional Testing
+
+TODO: overview of how to write and run functional tests with [yea](https://github.com/wandb/yea)
+and the [yea-wandb](https://github.com/wandb/yea-wandb) plugin.
+
+The `yea-wandb` plugin for `yea` uses copies of several components from `tests/utils` 
+(`artifact_emu.py`, `mock_requests.py`, and `mock_server.py`) 
+to provide a test environment for functional tests. Currently, we maintain a copy of those components in 
+`yea-wandb/src/yea_wandb`, so they need to be in sync.
+
+If you update one of those files, you need to:
+- While working on your contribution:
+  - Make a new branch (say, `shiny-new-branch`) in `yea-wandb` and pull in the new versions of the files.
+    Make sure to update the `yea-wandb` version.
+  - Point the client branch you are working on to this `yea-wandb` branch.
+    In `tox.ini`, search for `yea-wandb==<version>` and change it to 
+    `https://github.com/wandb/yea-wandb/archive/shiny-new-branch.zip`.
```

Comment:
nit: maybe use the git (ssh clone) format?
`git+https://github.com/wandb/yea-wandb.git@...#egg=yea-wandb`

Reply:
Whatever you think is better, I have no preference (and quite frankly don't understand the implications of the difference ;) -- feel free to commit your suggestion :)